### PR TITLE
[Wallets] Add wallet analytics

### PR DESCRIPTION
**Pull request by joaquim-verges**
## Problem solved

Enable wallet analytics for app developers

## Changes made

- [ ] Public API changes: `analytics` can be explicitely enabled/disabled on each wallet constructor
- [ ] Internal API changes: fire and forget analytics call on connection

## How to test

- [ ] Automated tests: link to unit test file
- [ ] Manual tests: step by step instructions on how to test

### CODE REVIEW

File: packages/wallets/src/evm/utils/analytics.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { fetch } from "cross-fetch";
+
+const ANALYTICS_ENDPOINT = "https://c.thirdweb-dev.com/event";
```
Comment (joaquim-verges) [inline packages/wallets/src/evm/utils/analytics.ts:3]: TODO change this to prod

File: packages/wallets/src/evm/utils/analytics.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { fetch } from "cross-fetch";
```
Comment (jnsdls) [inline packages/wallets/src/evm/utils/analytics.ts:1]: don't need `cross-fetch` anymore we no longer support any environment that doesn't have native fetch

**Pull request merged**